OpenSim/Simulation/Manager/Manager.h

```diff
@@ -221,17 +223,26 @@ class OSIMSIMULATION_API Manager
     // EXECUTION
     //--------------------------------------------------------------------------
     /**
+    * Initializes the Manager by creating and initializing the underlying 
+    * SimTK::TimeStepper. This must be called before calling 
+    * Manager::integrate() Subsequent changes to the State object passed in 
+    * here will not affect the simulation. Calling this function multiple 
+    * times with the same Manager will trigger an Exception.
```

Comment:
Since you have a separate `initialize(state)` why not create a new `TimeStepper` internally instead of forcing to recreate the `Manager`? Then we could use `initialize(state)`to update the state between integrations, which would be nice (e.g. #1662). We could make a note that this can be slow and it is more efficient to use an event handler to update the state or controls periodically.

Reply:
From testing before, this seemed to cause a problem. @chrisdembia mentioned it that it's "reinitializing" the Integrator that seemed to be a problem for some reason that we couldn't track down.